Installer/data/OpenCore/source.txt

```diff
@@ -1,2 +1,2 @@
-https://github.com/acidanthera/OpenCorePkg/releases/download/0.5.6/OpenCore-0.5.6-RELEASE.zip
-aefc2a120fa8a6265c88780c54bfb8ef68b741438f5dc27921ecb85cc800d760
\ No newline at end of file
+https://github.com/acidanthera/OpenCorePkg/releases/download/0.5.7/OpenCore-0.5.7-RELEASE.zip
```

Comment:
This is surely not going to pass the CI.

OpenCore 0.5.7 has a lot of breaking changes.

https://github.com/acidanthera/OpenCorePkg/blob/0.5.7/Docs/Differences/Differences.pdf